### ISSUE TITLE
What method are you using to measure the current used by the Pico?

### ISSUE THREAD

**Issue by garyexplains** [ghubcoder/PicoSleepDemo#3]
Hi,

Sorry for opening an issue, but I didn't know how else to contact you.

What method are you using to measure the current used by the Pico? A bench power supply? Multimeter?

Thanks 👍

**Comment by ghubcoder** [comment]
Yep just using a multimeter.... nothing fancy unfortunately ! Just setup in series with battery supply.
